### DISCUSSION TITLE
Tren yolcuları sayı sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#231] [Soru Cevap] [answered]
Site olarak 100 kişi bir tren yolculuğuna çıkmışız. 4 vagonu bulunan bu trende, 1. vagondan 2. vagona 16 yolcu geçerse, 2. vagondan 3. vagona 12 yolcu geçerse ve 3. vagondan 4. vagona 8 yolcu geçerse vagonlardaki yolcuların sayısı eşit oluyor. Başlangıçta vagonlarda kaç yolcu vardı?

**Comment by adaakoluk** [comment, marked as answer]
Vagonlardaki yolcuların sayısının eşit olması için her vagonda 25 kişi bulunmalı. (100/4) 1. vagondan 2. vagona 16 yolcu geçecekse 1. vagonda 41 yolcu olması gerek.(25+16 = 41) 2. vagona 16 yolcu geçtiğinde 25 yolcu olacaksa başlangıçta 9 yolcu vardır. (25-16) 3. vagona da 12 yolcu geçireceği için 2. vagonda toplam 21 yolcu vardır. (9+12) 3. vagonda bulunan yolcuları bulmak için 2. vagondan gelen yolcuları 25'den çıkarıp (25-12) 4. vagona geçireceği yolcuları toplamamız lazım. (13+8=21) 4. vagonda da 25 yolcu olabilmesi için 8 yolcu geçiyorsa başlangıçta 17 yolcu var demektir.(25-8)

Sonuç olarak: 
1. vagon: 41 
2. vagon: 21
3. vagon: 21
4. vagon: 17
